mne/tests/test_event.py

```diff
@@ -405,4 +412,30 @@ def test_define_events():
     assert_array_equal(true_lag_fill, lag_fill)
     assert_array_equal(true_lag_nofill, lag_nofill)
 
+
```

Comment:
And I think ``@testing.requires_testing_data`` decorator here will fix the other error.